docs/references/express/migrate-from-node-to-express.mdx

```diff
@@ -0,0 +1,104 @@
+---
```

Comment:
I think maybe putting this under upgrade guides might make more sense

Reply:
Aren't upgrade guides meant to be related to major Clerk upgrades? I think that we should keep it here since it's specific to the Clerk express package and how you can use it if you are already using the `@clerk/clerk-sdk-node` package.